pkg/api/v1/testkube/model_test_workflow_execution.go

```diff
@@ -28,7 +28,9 @@ type TestWorkflowExecution struct {
 	Signature []TestWorkflowSignature `json:"signature,omitempty"`
 	Result    *TestWorkflowResult     `json:"result,omitempty"`
 	// additional information from the steps, like referenced executed tests or artifacts
-	Output           []TestWorkflowOutput `json:"output,omitempty"`
+	Output []TestWorkflowOutput `json:"output,omitempty"`
+	// generated reports from the steps, like junit
+	Report           []TestWorkflowReport `json:"report,omitempty"`
```

Comment:
`reports`?

Reply:
Yep, should be plural. I am still not sure could we have other reports besides junit, or have this generic structure for report, where you have a `kind` field which can denote the report type.